metrics/lib/common.bash

```diff
@@ -42,6 +42,27 @@ function save_results(){
 	bash $LIB_DIR/send_results.sh -n "$1" -a "$2" -r "$3" -u "$4"
 }
 
+# This function checks existence of commands.
+# They can be received standalone standalone or as an array, e.g.
+#
+# cmds=(“cmd1” “cmd2”)
+# check_cmds "${cmd[@]}"
+function check_cmds()
+{
+	set +e
```

Comment:
OK, I'm going to ask for some input on shell-ness here from @MarioCarrilloA and @bwarden (coz otherwise I just have to go look it up, and you guys probably already know....)
the `set -e`, does that just have effect in this func - are funcs run in sub-shells? (I think they are).
If yes, then this is fine, but we need to remove the `set -e` at the bottom of the func as it is redundant.
If no, then we should not be blatantly doing the `set -e` at the end of the func, as you may have just turned on `-e` for a script that did not have it set when it called this func.
So, give me your input, and let's get this fixed ;-)